Solutions/CohesitySecurity/Tests/Alert.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+import json
+
+
+class Alert:
+    def __init__(self, json_str):
+        data = json.loads(json_str)[0]
+        self.jobId = None
+        self.clusterId = data['clusterId']
+        self.clusterIncarnationId = None
+        for prop in data['propertyList']:
+            if prop['key'] == 'jobId':
+                self.jobId = prop['value']
+            elif prop['key'] == 'clusterIncarnationId':
+                self.clusterIncarnationId = prop['value']
+        self.protectionGroupId = f"{self.clusterId}:{self.clusterIncarnationId}:{self.jobId}"
+
+    def get_jobId(self):
```

Comment:
Why is the name 'get_jobId' a mix of snake case and camel case? Same for the other function names as well.

Reply:
done